test/spec/oauthUtil.js

```diff
@@ -96,6 +96,71 @@ define(function(require) {
         }));
       }
     });
+    util.itMakesCorrectRequestResponse({
+      title: 'caches response in sessionStorage if localStorage isn\'t available',
+      setup: {
+        beforeClient: function() {
+          oauthUtilHelpers.mockLocalStorageError();
+        },
+        calls: [
+          {
+            request: {
+              method: 'get',
+              uri: '/.well-known/openid-configuration'
+            },
+            response: 'well-known'
+          }
+        ],
+        time: 1449699929
+      },
+      execute: function(test) {
+        sessionStorage.clear();
+        return oauthUtil.getWellKnown(test.oa)
+        .then(function() {
+          return oauthUtil.getWellKnown(test.oa);
```

Comment:
It's not necessary. I didn't remove it after a copy/paste from another test. Will remove